ios/PolyPodApp/Dependencies/PolyPodCore.xcframework/ios-arm64/Headers/polypodcore.h

```diff
@@ -0,0 +1,64 @@
+#include <stdarg.h>
```

Comment:
We have to duplicate the headers for different architectures?
If this is generated, that would be another good reason for *not* including generated code in the repo...

Reply:
Headers are duplicated by XCFrameworks, based on its internal structure.